gsoc/2018.md

```diff
@@ -81,6 +81,37 @@ Scalafix is a fast moving project, so you will be working in a codebase with mul
 
 Supervised by [@olafurpg](https://github.com/olafurpg)
 
+### Fox, a documentation engine for Scala
+
+Documentation may be consumed and written in many ways. Library authors can
+write tutorials, in-depth guides, or Scala docs; and users can consume them in
+PDF files, online websites or ebook readers. There are many hidden requirements
+in writing good documentation and, given how difficult it is to get it right,
+it is a task that many developers eventually ignore or, at best, procrastinate
+at.
+
+This project calls for the creation of a flexible documentation engine that
+aims to simplify writing documentation for library authors, so that they can
+focus on the inherent difficulty of writing good docs.
+
+The scope of the project is to continue the development of an existing
+prototype, called `fox`. The project can start small with simple changes to get
+familiar with the codebase, and gradually step up as progress happens.
+
+Your end goal will be to implement the following features:
+
+1. A [Flexmark][] integration to generate PDF and ebooks (epub, mobi) using
+   Pandoc.
+1. A series of modifications to the default website template that:
```

Comment:
Another nice-to-have feature I'd love in a doctool: local server that refreshes browser on edit, like https://www.gatsbyjs.org/, http://www.mkdocs.org/ and https://gohugo.io/ have